kitsune/users/models.py

```diff
@@ -69,6 +69,10 @@ class Profile(ModelBase, SearchMixin):
                             verbose_name=_lazy(u'City'))
     locale = LocaleField(default=settings.LANGUAGE_CODE,
                          verbose_name=_lazy(u'Preferred language'))
+    first_answer_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first answer contribution email.'))
+    first_l10n_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first revision contribution email.'))
```

Comment:
`verbose_name` is the longer name for the filed name. So this would be something like "First revision contribution email". It gets autogenerated by Django, so setting it overrides the default.

`verbose_name` and `help_text` show up in a few places:

1. any forms that are based on this model
2. the admin
3. autogenerated documentation

https://docs.djangoproject.com/en/1.8/topics/db/models/#verbose-field-names
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.Field.help_text

Generally, I've not been setting `verbose_name` (the name is usually sufficient), but I have been setting `help_text` especially when the value causes interesting behavior or has some property like "can use Markdown".